src/encoded/static/components/static-pages/placeholders/BasicCarousel.js

```diff
@@ -7,11 +7,12 @@ import Carousel from 'nuka-carousel';
 
 /** @see https://www.npmjs.com/package/nuka-carousel for documentation of all props/options allowed */
 const defaultCarouselOptions = {
-    'initialSlideHeight' : 540,
-    'initialSlideWidth' : 720,
+    //'initialSlideHeight' : 540,
+    //'initialSlideWidth' : 720,
     'autoplay': false,
     'autoplayInterval': 5000,
     'cellSpacing': 20,
+    'cellAlign': "center",
     'heightMode' : "max",
```

Comment:
@alexkb0009  carousel prop heightMode: 'max' hides/overrides the width and height props. may remove it from default configuration?

Reply:
Maybe it might be worth setting if slideHeight and slideWidth are undefined (idk / random thought).